File: CODE FILES/Backend/MyApiary/Resource.Api/Models/Beehive.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace Resource.Api.Models
{
    public class Beehive
    {
        [Required]
        public int Id { get; set; }

        [Required]
        public string Name { get; set; }

        [Required]
        public bool Alarm { get; set; }

        [Required]
        public float SensorTemperature { get; set; }

        [Required]
        public float SensorHumidity { get; set; }

        [Required]
        public float SensorNoise { get; set; }

    }
}

File: CODE FILES/Mobile/MyApiary/app/src/main/java/projects/myapiary/models/Beehive.kt
package projects.myapiary.models

import com.google.gson.annotations.SerializedName

data class Beehive(
    @SerializedName("id")
    val Id: Int,
    @SerializedName("name")
    val Name: String,
    @SerializedName("alarm")
    var Alarm: Boolean,
    @SerializedName("sensorTemperature")
    var SensorTemperature: Float,
    @SerializedName("sensorHumidity")
    val SensorHumidity: Float,
    @SerializedName("sensorNoise")
    val SensorNoise: Float
)
